src/components/Popover/index.native.js

```diff
@@ -13,20 +13,17 @@ const propTypes = {
  * This is a convenience wrapper around the Modal component for a responsive Popover.
  * On small screen widths, it uses BottomDocked modal type, and a Popover type on wide screen widths.
  */
-const Popover = (props) => {
-    const propsWithoutAnimation = _.omit(props, ['animationIn', 'animationOut', 'popoverAnchorPosition', 'disableAnimation']);
-    return (
-        <Modal
-            type={CONST.MODAL.MODAL_TYPE.BOTTOM_DOCKED}
-            // eslint-disable-next-line react/jsx-props-no-spreading
-            {...propsWithoutAnimation}
-
-            // Mobile will always has fullscreen menu
-            // eslint-disable-next-line react/jsx-props-no-multi-spaces
-            fullscreen
-        />
-    );
-};
+const Popover = props => (
+    <Modal
+        type={props.fromSidebarMediumScreen ? CONST.MODAL.MODAL_TYPE.POPOVER : CONST.MODAL.MODAL_TYPE.BOTTOM_DOCKED}
+        popoverAnchorPosition={props.isSmallScreenWidth ? undefined : props.anchorPosition}
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...props}
+        fullscreen={props.isSmallScreenWidth ? true : props.fullscreen}
+        animationIn={props.isSmallScreenWidth ? undefined : props.animationIn}
+        animationOut={props.isSmallScreenWidth ? undefined : props.animationOut}
```

Comment:
Any reason we're forcing no animation on small screens? We should be keeping the default we currently have for native (iOS/Android) where it slides up from the bottom

Reply:
I'll be working fixing this animation regressions